src/ucp/api/ucp.h

```diff
@@ -660,16 +664,30 @@ typedef struct ucp_ep_params {
     uint64_t                field_mask;
 
     /**
-     * Destination address; the address must be obtained using
-     * @ref ucp_worker_get_address.
-     * This field is mandatory for filling (along with its corresponding bit
+     * Destination address; if a connection manager isn't used,
```

Comment:
need to drop "connection manager" terminology from API; this is implementation detail. Need to use "client/server connection establishment" or "active/passive connection establishment"